quickstarts/Safety.ipynb

```diff
@@ -61,7 +61,7 @@
       "source": [
```

Comment:
<blockquote><strong>Important:</strong> If the unsafety degree of your prompt is too high</blockquote><p> </p><p>I don't think this is a question of degree, I think it's more just that there are some filters we can't legally turn off.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Fuck . Shit . Damn . Wtf . What a hell</p>